.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "[python]": {
+    // [comments, max line length, wrap line length]
+    // Black does not wrap comments.
+    "editor.rulers": [80, 88, 120],
+    "editor.wordWrap": "wordWrapColumn",
+    "editor.wordWrapColumn": 120,
```

Comment:
Does this wrap lines at 120 words? I thought we were going with 88?

Reply:
Black will only wrap code at 88 characters, but it doesn't wrap comments. VSCode is configured to 120 as fallback for wrapping comments.

> Black currently doesn't wrap long string literals or merge string literals that happen to be on the same line. [...] It would require modifying the AST which isn't 100% safe and has a bunch of edge cases to be dealt with.
https://github.com/psf/black/issues/1331